kafka/producer/simple.py

```diff
@@ -41,19 +42,24 @@ class SimpleProducer(Producer):
     def __init__(self, client, async=False,
                  req_acks=Producer.ACK_AFTER_LOCAL_WRITE,
                  ack_timeout=Producer.DEFAULT_ACK_TIMEOUT,
-                 codec=None,
+                 codec=CODEC_NONE,
                  batch_send=False,
                  batch_send_every_n=BATCH_SEND_MSG_COUNT,
                  batch_send_every_t=BATCH_SEND_DEFAULT_INTERVAL,
-                 random_start=True):
+                 random_start=True,
+                 maxsize=0):
```

Comment:
You probably mean `maxsize=None` here, and let the `Producer` class determine the default.

Right now you have the default in 2 places, both here and in `ASYNC_QUEUE_MAXSIZE`.